.drone.star

```diff
@@ -73,7 +73,7 @@ config = {
         "oC10": {
             "db": "mysql:5.5",
             "earlyFail": True,
-            "skip": False,
+            "skip": True,
```

Comment:
```suggestion
            "skip": False,
```

Reply:
i am actually testing a scenario on drone so this is just temporary.

Reply:
ok, sorry I thought those were leftovers. for future reference, if you do something like this and the PR is in `review in progress` please leave a comment on the changes saying that they will be reverted 😅